strings/uctypedump.c

```diff
@@ -225,7 +225,7 @@ int main(int ac, char ** av)
   {
     char plane_name[128]="NULL";
     if(uctype[plane].ctype){
-      sprintf(plane_name,"uctype_page%02X",(uint) plane);
+      snprintf(plane_name,sizeof(plane_name),"uctype_page%02X",(uint) plane);
```

Comment:
Not needed, 128 bytes allocated, at most 15 used here.

Reply:
The whole point of this is that individual humans don't need to go through and reason about the allocated-vs-used space for each and every individual call.

When we miss or misunderstand one, that's how C memory bugs happen.

Reply:
Modern compilers however pick this up quite well.